mvrx-rxjava2/gradle.properties

```diff
@@ -1,4 +1,4 @@
-POM_NAME=MvRx
-POM_ARTIFACT_ID=mvrx-rxjava2
+POM_NAME=Mavericks
+POM_ARTIFACT_ID=mavericks-rxjava2
```

Comment:
should this pr include updates to the docs that reference these artifacts as well?